tests/nimble_autoconn_gnrc/Makefile

```diff
@@ -0,0 +1,23 @@
+BOARD ?= nrf52dk
+include ../Makefile.tests_common
+
+# include shell support
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+# Include GNRC and RPL
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_ipv6_router_default
+USEMODULE += gnrc_icmpv6_echo
+USEMODULE += auto_init_gnrc_rpl
```

Comment:
Is `rpl` needed?

Reply:
Yes, it is useful and makes sense to include - testing multi-hop IP without a routing protocol is quite pointless :-)

Reply:
OK!